Firestore/Swift/Source/Result/Query+Result.swift

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import FirebaseFirestore
+
+@available(swift 5.0)
+extension Query {
+  /// Reads the documents matching this query.
+  ///
+  /// - Parameters:
+  ///   - source: The `FirestoreSource` where the results should be fetched:
+  ///     - `default`:  Fetch from the server and, if it fails, the cache.
+  ///     - `server`:  Fetch from the server only.
+  ///     - `cache`:  Fetch from the cache only.
+  ///   - completion: The closure to execute on receipt of a result.
+  ///   - result: The result of request. On success it contains the `QuerySnapshot`, otherwise an
+  ///   `Error`.
+  func getDocuments(source: FirestoreSource = .default,
+                    completion: @escaping (_ result: Result<QuerySnapshot, Error>) -> Void) {
+    getDocuments(source: source, completion: mapResultCompletion(completion))
+  }
+
+  /// Attaches a listener for this `Query` events.
+  ///
+  /// - Parameters:
+  ///   - includeMetadataChanges: Whether metadata-only changes (i.e. only
+  ///   `DocumentSnapshot.metadata` changed) should trigger snapshot events.
+  ///   - listenerHandler: The closure to execute on receipt of a result.
+  ///   - result: The result of request. On success it contains the `QuerySnapshot`, otherwise an
+  ///   `Error`.
+  /// - Returns: The `ListenerRegistration` that can be used to remove this listener.
+  func addSnapshotListener(includeMetadataChanges: Bool = false,
+                           listenerHandler: @escaping (_ result: Result<QuerySnapshot, Error>)
```

Comment:
Suggestion: s/listenerHandler/handler to match the API update above

Reply:
Do you mean this:
```suggestion
                           listener handler: @escaping (_ result: Result<QuerySnapshot, Error>)
```
or this
```suggestion
                           listener: @escaping (_ result: Result<QuerySnapshot, Error>)
```

Reply:
The latter (updated my comment as well)

Reply:
Done!